src/api/spec/db/data/remove_notifications_for_missing_notifiables_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+require Rails.root.join('db/data/20210205131609_remove_notifications_for_missing_notifiables.rb')
+RSpec.describe RemoveNotificationsForMissingNotifiables, type: :migration do
+  describe 'up' do
+    let!(:notification1) { create(:notification, :comment_for_request) }
+    let!(:notification2) { create(:notification, :comment_for_request) }
+
+    before do
+      # Simulate the wrong behaviour we had before. When a BsRequest was removed,
+      # all the comments were removed as well, but not their associated notifications.
+      # That happended because of a wrong association, dependent option: `delete_all` instead of `destroy`.
```

Comment:
Really not a blocker, just a small typo :)

```suggestion
      # That happened because of a wrong association, dependent option: `delete_all` instead of `destroy`.
```

Reply:
Ohh, thanks! Corrected!